cont.c

```diff
@@ -433,6 +433,12 @@ fiber_pool_allocate_memory(size_t * count, size_t stride)
             *count = (*count) >> 1;
         }
         else {
+#if defined(MADV_FREE_REUSE)
```

Comment:
Is `MADV_FREE_REUSE` a MacOS specific thing? Because it's different from below (`MADV_FREE_REUSABLE`).

Reply:
Yes it s Darwin specific.

Reply:
We should add a comment to explain this.

Reply:
I checked macOS man page for `madvise` and could not find reference to `MADV_FREE_REUSE`. Are you sure it's valid?